### DISCUSSION TITLE
script freezes

### DISCUSSION THREAD

**Discussion by GreatHolyCow** [jertel/vuegraf#37] [Q&A] [answered]
I have this all setup originally on an older RaspberryPi, I upgraded to a new RaspberryPi. Installed everything fresh on the new one and it runs for like 5-10mins then just pauses stops doing anything no error, no crash, nothing. I have since tried to see where its hanging by doing adding  "-m trace --trace" and now it wont hang/pause (edit froze 6hrs later).... The exact same config/script works perfectly on the old pi but doesn't seem to work well on the new pi unless at this point I leave the trace on... Is there something else I can to do debug/find what's different between the systems? I installed all the same requirements on both systems.

**Comment by jertel** [comment, marked as answer]
If i was in your shoes I would probably do a tcpdump on the Pi to see if it is waiting for a network response from either the influx db or from Emporia.

**Reply by GreatHolyCow** [reply]
Strange thing is now after just a reboot and turning off the "-m trace --trace" its been running for 36hrs straight so far... nothings changed no configs edited no packages installed/removed...

**Reply by jertel** [reply]
Yes that is odd. Keep us posted.
